tools/dependency/utils.py

```diff
@@ -14,16 +15,16 @@ def LoadModule(name, path):
   return module
 
 
-envoy_repository_locations = LoadModule('envoy_repository_locations',
```

Comment:
these are imported into other modules (or at least one)

i would like to figure why this didnt break CI